std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -450,6 +450,7 @@ struct KRRegion(ParentAllocator = NullAllocator)
             // Insert right after root
             n.next = root.next;
             root.next = n;
+            ensureCorrectSize();
```

Comment:
This seems unnecessary - there are at least two nodes here and the problem only occurs when there is exactly one node. Correct?